app/serializers/source_data/product_serializer.rb

```diff
@@ -18,6 +18,7 @@ def deserialize(payload, product_type)
       if product_type == "ShopifyProduct"
         properties = payload.delete("attributes")
         payload["properties"] = properties
+        payload["product_category_id"] = "gid://shopify/TaxonomyCategory/#{payload["product_category_id"]}"
```

Comment:
_nit_ Would be nice to have a reusable method for this. Like `Category.gid`

Does this gid work for the old taxonomy @elsom25?

Reply:
Ah, I see there's already a similar comment thread: https://github.com/Shopify/product-taxonomy/pull/134/files#r1563212184